docs/usage/wasm-constraints.md

```diff
@@ -10,13 +10,18 @@ or via [patches](https://github.com/pyodide/pyodide/tree/main/cpython/patches).
 
 ### Optional modules
 
-The following stdlib modules are included by default, however
-they can be excluded with `loadPyodide({ fullStdLib : false })`.
-Individual modules can then be loaded as necessary using
-{any}`pyodide.loadPackage`,
+The following stdlib modules are unvendored by default,
+in order to reduce initial download size of Python distribution.
+You can load all unvendored stdlib modules
+when initializing Pyodide with, `loadPyodide({ fullStdLib : true })`.
```

Comment:
"however this will have a significant implement on the download size.
Instead, it is better to load individual modules as needed using ... "